controller/controller.js

```diff
@@ -35,13 +36,20 @@ function getCommentsByReviewId(request, response, next) {
     }).catch(next)
 }
 
+
 function postCommentByReviewId(request, response, next) {
     const {review_id} = request.params
     Promise.all([fetchReviewsByReviewId(review_id), addCommentByReviewId(review_id, request.body)]).then((result) => {
         response.status(201).send({comment: result[1]})
     }).catch(next)
 
+function patchReview(request, response, next) {
+    const {review_id} = request.params
+    const {inc_votes} = request.body
```

Comment:
If this value were missing because of a bad request, would need to send a reject the promise with a 400 🔍